src/WalkingModule/src/Module.cpp

```diff
@@ -598,13 +751,423 @@ bool WalkingModule::updateModule()
         }
 
         // evaluate 3D-LIPM reference signal
-        m_stableDCMModel->setInput(m_DCMPositionDesired.front());
+        if(m_useStepAdaptation)
+        {
+            if (!m_leftInContact.front() || !m_rightInContact.front())
+            {
+                if(!m_leftInContact.front()){
+                    if (!feetTrajectorySmoother(m_adaptatedFootLeftTransform,m_leftTrajectory.front(),m_smoothedFootLeftTransform,m_adaptatedFootLeftTwist,m_leftTwistTrajectory.front(),m_smoothedFootLeftTwist)) {
+                        yError()<<"the Left Foot trajectory smoother can not evaluate the smoothed Trajectoy!";
```

Comment:
Can you deal with the formatting of these error messages? Also, there is typo ``Trajectoy``. The same below.

Reply:
Done!